characters.py

```diff
@@ -0,0 +1,13 @@
+"""
+The `Player` class represents a player in the game. It has an inventory and a dialogue.
+
+The `NPC` class represents a non-player character in the game. It has dialogue.
+"""
+class Player:
```

Comment:
Missing methods here!! Your Player class needs at least two methods - which you have, just not in the class! But you could add things like "talk to character", "introduce myself" etc!